srcs/juloo.keyboard2/Config.java

```diff
@@ -385,14 +385,57 @@ private int getThemeId(Resources res, String theme_name)
   {
     switch (theme_name)
```

Comment:
This switch statement needs to go! Linear lookup is the best way, see the layout lookup as an example (it uses a third generated array containing the resources ids) https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/prefs/LayoutsPreference.java#L53